src/sentry/static/sentry/app/routes.jsx

```diff
@@ -576,6 +576,27 @@ function routes() {
           component={errorHandler(LazyLoad)}
         />
       </Route>
+      <Route name="Developer Settings" path="developer-settings/">
+        <IndexRoute
+          componentPromise={() =>
+            import(/*webpackChunkName:"OrganizationDeveloperSettings"*/ './views/settings/organizationDeveloperSettings')}
+          component={errorHandler(LazyLoad)}
+        />
+        <Route
```

Comment:
What you have here is good (and clear) - I *think* you can nest the routes like below to remove some duplication. But this is more fyi than asking for a change.

```
        <Route
          componentPromise={() =>
            import(/*webpackChunkName: "sentryApplicationDetails"*/ './views/settings/organizationDeveloperSettings/sentryApplicationDetails')}
          component={errorHandler(LazyLoad)}
        >
          <Route
            name="New Application"
            path="new/"
          />
          
          <Route
            name="Edit Application"
            path=":appSlug/"
          />
        </Route>
```

Reply:
cool cool cool will keep in mind for future :)